bin/install-site.sh

```diff
@@ -307,6 +323,15 @@ Pin-Priority: 50
 EOF
     fi
 
+    if [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"jessie" ]
```

Comment:
The only packages which I remember being different are the ruby ones - `ruby2.1` and `ruby2.1-dev` instead of the 1.8 or 1.9 builds (and therefore presumably `rubygems` and probably `bundler` are also affected)